hexa/workspaces/schema.py

```diff
@@ -109,4 +123,51 @@ def resolve_delete_workspace(_, info, **kwargs):
         return {"success": False, "errors": ["PERMISSION_DENIED"]}
 
 
+@worskspace_mutations.field("createWorkspaceMember")
```

Comment:
I'd prefer "inviteWorkspaceMember" WDYT ?

Reply:
Yes, it's better